src/webview/html/messageHeaderAsHtml.js

```diff
@@ -57,45 +60,49 @@ export default (
     const topicHtml = renderSubject(item);
 
     return template`
-<div class="header-wrapper stream-header" data-msg-id="${item.id}">
+<div class="header-wrapper header stream-header" data-msg-id="${item.id}">
   <div class="header stream-text"
        style="color: ${textColor};
               background: ${backgroundColor}"
        data-narrow="${streamNarrowStr}">
     # ${item.display_recipient}
   </div>
-  <div class="header topic-text" data-narrow="${topicNarrowStr}">
-    $!${topicHtml}
+  <div class="header-wrapper header topic-content" data-narrow="${topicNarrowStr}">
```

Comment:
Why the nested `header-wrapper`, and the nested `header`? That seems likely to cause confusion and/or a bug, if it doesn't cause a live bug already.

Reply:
Also `topic-content` is *definitely* going to be confusing if what it really means is "topic plus date". The date isn't really anything to do with the topic, so `topic-content` doesn't give a hint of that.

If you want a common class for "topic + date" in this and the other case, you could if nothing else call it "topic-and-date". But before that, I would ask why they're grouped together in the first place. :slightly_smiling_face: Why not add the "date" div as a sibling of both the `stream-text` and the `topic-text`?

Reply:
My original intention of grouping them as separate div's was to make it clear that there will be different classes to represent both the text and the date, and I'll admit my inexperience with CSS; I don't know how to not bring two divs and still maintain the flexbox with `justify-content: space-between;` The date manages to overflow no matter what changes I make to the CSS to prevent it from doing so. Making two divs solves this problem, so I went with it.
<img width="367" alt="Screenshot 2019-04-28 at 11 54 13 AM" src="https://user-images.githubusercontent.com/33200024/56859725-8ea54d00-69ac-11e9-8ee6-9ee5221453d1.png">

Suggestions about changes to the CSS are welcome! 😄

Reply:
The `header-wrapper` and `header` nesting was definitely redundant (although no observable bugs could be found by me) and has been taken care of, thanks! Also changed the name for `topic-content` to `topic-header` in order to correctly represent what type of narrow the header belongs to (like `stream-header` and `private-header` represent their narrows.)